dataland-frontend/tests/e2e/utils/Converters.ts

```diff
@@ -0,0 +1,5 @@
+export function convertStringToQueryParamFormat(inputString: string): string {
+  const stringInQueryParamFormat = inputString.replace(" ", "+")
```

Comment:
spaces and special characters are not allowed in urls and are replaced with other characters. so if we check the url for a specific string we need to replace spaces before or it will not match

Reply:
We have some tests that expect a certain url on a certain page.
Something like "expect url to equal           **/companies?sector=fashion".

Now, our sectors are randomly generated by a faker, right?
So sometimes it happens, that the faker generates a sector with a space in it. E.g. "web services".

Now our test would do   "expect url to equal           **/companies?sector=web services".

But in URLs, spaces are forbidden.  They are replaced by a "+".

So the test should check "expect url to equal           **/companies?sector=web+services". 
insted, right?

To assure that, we use this converter to replace spaces by "+" before we use a string as expected value inside a url.